rules/sinks/leakages/logs/java.yaml

```diff
@@ -29,3 +29,9 @@ sinks:
     patterns: 
       - "(?i)(java.io.PrintStream|android.util.(?:Log){0,1}Printer).(?:print|println|write).*"
     tags:
+
+  - id: Leakages.Log.Equals
```

Comment:
@hiaashna I guess this rule is not needed, as per my understanding this is used to compare 2 log levels. What you feel?

Reply:
@khemrajrathore  yeah, I feel the same and I think whichever logger is supposed to expose any personal info have already been covered.

Reply:
@khemrajrathore I have removed log equals and added a log that contain IP address